README.md

```diff
@@ -8,7 +8,7 @@ Or [Blockchain](<https://en.wikipedia.org/wiki/Blockchain_(database)>), for shor
 
 [![version](https://img.shields.io/github/tag/tendermint/tendermint.svg)](https://github.com/tendermint/tendermint/releases/latest)
 [![API Reference](https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667)](https://pkg.go.dev/github.com/tendermint/tendermint)
-[![Go version](https://img.shields.io/badge/go-1.14-blue.svg)](https://github.com/moovweb/gvm)
+[![Go version](https://img.shields.io/badge/go-1.15-blue.svg)](https://github.com/moovweb/gvm)
```

Comment:
Should this be in its own PR since its breaking?

Reply:
what it breaks exactly?

Reply:
If this PR is back-ported as is, users will have to bump their go version. I would consider this a breaking change since they can't restart their node without this.

Reply:
Yeah, I'd agree, let's split out the Go bump to a separate PR.

Reply:
then we won't be able to reset the timer. but I guess it's fine.